index.html

```diff
@@ -3054,45 +3040,57 @@ <h3>Production</h3>
 <a data-cite="INFRA#boolean">boolean</a>
               </td>
               <td>
-<a data-cite="RFC8949#section-3.3">CBOR simple value</a> (major type 7,
-subtype 24) with a simple value of 21 (True) or 20 (False)
+A <a data-cite="RFC8949#section-3.3">CBOR simple value</a> (major type 7,
+subtype 24) with a simple value of 21 (true) or 20 (false).
               </td>
             </tr>
             <tr>
               <td>
 <a data-cite="INFRA#null">null</a>
               </td>
               <td>
-<a data-cite="RFC8949#section-3.3">CBOR simple value</a> (major type 7,
-subtype 24) with a simple value of 22 (Null)
+A <a data-cite="RFC8949#section-3.3">CBOR simple value</a> (major type 7,
+subtype 24) with a simple value of 22 (null).
               </td>
             </tr>
           </tbody>
         </table>
 
-        <ul>
+        <p>
+In addition to the data type production rules above, the following rules apply
+for <a>conforming producers</a> that serialize CBOR <a>representations</a>:
+        </p>
+        <ol>
           <li>
-Other Data Types not otherewise defined in the <a href="#data-model">data model</a> MUST be
-represented as a <a data-cite="RFC8949#section-3.1">CBOR
-String type</a> (major type 3).
+Indefinite-length items are not allowed and MUST be made <a
+data-cite="RFC8949#section-3.2.2">CBOR definite length.</a>
           </li>
           <li>
-Indefinite-length items are not allowed and MUST be made <a
-data-cite="RFC8949#section-3.2.2">CBOR definite
-length.</a>
+All CBOR tags MUST be retained regardless of whether they are optional.
           </li>
-        </ul>
+          <li>
+All four <a data-cite="RFC8949#section-4.2.1">Canonical CBOR rules</a>
+listed in [[RFC8949]] MUST be applied to all relevant data types.
+          </li>
+        </ol>
 
         <p>
-In addition to the production rules in this section, implementations
-MUST implement the four
-<a data-cite="RFC8949#section-4.2.1">Canonical CBOR rules</a>
-listed in [[RFC8949]]. All CBOR tags that appear in a message MUST be retained
-regardless of whether they are optional.
+All entries of a <a>DID document</a> MUST be included in the root <a
+data-cite="RFC8949#section-3.1">CBOR map</a> (major type 5). Entries MAY contain
+additional data substructures subject to the value representation rules in the
+list above. When serializing a <a>DID document</a> in the CBOR
```

Comment:
```suggestion
list above. When serializing a <a>DID document</a> to its CBOR
```

Reply:
`its` might be `a` if there's more than one potential CBOR representation of a single state of a DID document... but I think there's only one.

Reply:
Yeah, the language here is tricky. There are at least two CBOR-based representations... Vanilla CBOR (this spec) and CBOR-LD (a future state, but with an implementation and DID document compression test). We are writing this in the CBOR section, so hopefully the reader understands that the "CBOR" we're talking about is "the CBOR representation described in this section".

All that said, I'm merging THIS suggestion.